ui/app/components/secret-engine/configure-aws.ts

```diff
@@ -43,11 +44,34 @@ interface Args {
   leaseConfig: LeaseConfigModel;
   rootConfig: RootConfigModel;
   backendPath: string;
+  issuer?: string;
+}
+
+class IssuerConfig {
+  original = '';
+  @tracked issuer = '';
+  @tracked noRead = false;
+  init(initialIssuer: string | undefined) {
```

Comment:
I was reading a little about `init()` vs `constructor()` is there a reason you opted to use `init()` here?

Reply:
Since I need to pass in an arg, I wasn't sure if that was going to be available on constructor. I can try it though

Reply:
Just a guess here, but because she calls the `init` method later. You initialize the class, and init gets called, but you can also then call init outside the class to isolate calling that specific method? Learning alongside you.

Reply:
Oh no need to change it, I was just curious if there was a reason! I can read up more on the pros/cons of each one. I do know that you can still pass args using constructor, I used it in the kv patch form.

Reply:
Feel free to share what you find! Since the constructor can be called with the arg I like that, but open to pros/cons